ui/lib/apps/SlowQuery/pages/List/index.tsx

```diff
@@ -107,11 +100,11 @@ function List() {
           </Space>
 
           <Space>
-            {columns.length > 0 && (
+            {tableColumns.length > 0 && (
               <ColumnsSelector
-                columns={columns}
+                columns={tableColumns}
                 visibleColumnKeys={visibleColumnKeys}
-                resetColumnKeys={defSlowQueryColumnKeys}
+                resetColumnKeys={DEF_SLOW_QUERY_COLUMN_KEYS}
```

Comment:
should this be called `defaultVisibleColumnKeys` instead?

Reply:
Done!